packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -161,110 +211,140 @@ body {
         }
       }
     }
+    &:focus-within {
+      border-color: var(--aa-primary-color);
+      box-shadow: var(--aa-primary-color-alpha) 0 0 0 2px,
+        inset var(--aa-primary-color-alpha) 0 0 0 2px;
+      outline: currentColor none medium;
+    }
   }
 }
 
 // ----------------
-// Panel
+// 4. Panel
 // ----------------
-
 .aa-Panel {
-  position: absolute;
-  // reset
-  * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-  }
-  button {
-    appearance: none;
-    background: none;
-    border: 0;
-  }
+  @extend %init;
 
-  &::after {
-    background-image: linear-gradient(
-      var(--aa-background-color-alpha-0),
-      var(--aa-background-color)
-    );
-    border-radius: 3px;
-    bottom: 0;
-    content: '';
-    height: 0.75em;
-    pointer-events: none;
-    position: absolute;
-    width: 100%;
-    z-index: 99;
+  background-color: var(--aa-background-color);
+  border-radius: calc(var(--aa-spacing) / 4);
+  box-shadow: var(--aa-panel-shadow);
+  margin: 8px 0 0;
+  overflow: hidden;
+  position: absolute;
+  transition: opacity 200ms ease-in, filter 200ms ease-in;
+  @media screen and (prefers-reduced-motion) {
+    transition: none;
   }
-  .aa-PanelLayout {
-    background-color: var(--aa-background-color);
-    border-radius: 3px;
-    box-shadow: var(--aa-panel-shadow);
+  @at-root .aa-PanelLayout {
     height: 100%;
-    margin-top: var(--aa-spacing-half);
+    margin: 0;
     max-height: var(--aa-panel-max-height);
-    padding-bottom: var(--aa-spacing-half);
-    padding-top: var(--aa-spacing-half);
+    overflow-y: auto;
+    padding: 0;
+    padding: var(--aa-spacing-half) var(--aa-spacing) var(--aa-spacing)
+      var(--aa-spacing-half);
     position: relative;
     text-align: left;
 
-    .aa-PanelLayoutResults {
-      overflow-x: hidden;
-      overflow-y: auto;
-      width: 50%;
+    &.aa-PanelLayout-cols-two--golden {
+      display: grid;
+      grid-template-columns: 39.2% auto;
+      overflow: hidden;
+      padding: 0;
     }
-    .aa-PanelLayoutPreview {
-      border-left: solid 1px var(--aa-selected-color);
-      flex-shrink: 1;
-      max-width: 50%;
+    &.aa-PanelLayout-cols-two {
+      display: grid;
+      grid-template-columns: repeat(2, minmax(0, 1fr));
       overflow: hidden;
+      padding: 0;
+    }
+    &.aa-PanelLayout-cols-three {
+      display: grid;
+      grid-template-columns: repeat(3, minmax(0, 1fr));
+      overflow: hidden;
+      padding: 0;
+    }
+
+    .aa-Panel--Scrollable {
```

Comment:
Are modifiers capitalized? In this case there's a clash with class names like `aa-Panel--stalled`.

Reply:
See ed288f3d0886d818d5654cc3f91ccc75d072d230.